pkg/cgroups/manager/manager.go

```diff
@@ -75,15 +76,15 @@ type cgroupManager struct {
 type PodMetadata struct {
 	Name      string
 	Namespace string
-	IPs       []string
+	IPs       []net.IP
```

Comment:
Just adding a comment that this should be a `netip.Addr`. That way we won't accidentally merge.